awx/ui/client/lib/components/input/text.partial.html

```diff
@@ -1,15 +1,42 @@
 <div class="col-sm-{{::col}} at-InputContainer">
     <div class="form-group at-u-flat">
         <at-input-label></at-input-label>
-
-        <input type="text" class="form-control at-Input" 
-               ng-class="{ 'at-Input--rejected': state._rejected }"
-               ng-model="state._value"
-               ng-attr-maxlength="{{ state.max_length || undefined }}"
-               ng-attr-tabindex="{{ tab || undefined }}"
-               ng-attr-placeholder="{{::state._placeholder || undefined }}"
-               ng-disabled="state._disabled || form.disabled" />
-
+        <div ng-if="state.tagMode" class="input-group">
+          <span class="input-group-btn input-group-prepend">
+              <button
+                class="btn at-ButtonHollow--default at-Input-button"
+                ng-disabled="state._disabled || form.disabled"
+                ng-click="vm.onLookupClick()">
+                  <i class="fa fa-search"></i>
+              </button>
+          </span>
+          <span ng-if="state.asTag" class="form-control at-Input">
+            <div class="at-InputTagContainer">
+              <at-tag
+                ng-show="state._tagValue"
+                icon="external"
+                tag="state._tagValue"
+                remove-tag="state._onRemoveTag(state)"
+              />
+            </div>
+          </span>
+          <input ng-if="!state.asTag" type="text" class="form-control at-Input"
+            ng-class="{ 'at-Input--rejected': state._rejected }"
+            ng-model="state._value"
+            ng-attr-maxlength="{{ state.max_length || undefined }}"
+            ng-attr-tabindex="{{ tab || undefined }}"
+            ng-attr-placeholder="{{::state._placeholder || undefined }}"
+            ng-disabled="state._disabled || form.disabled"
+          />
+        </div>
+        <input ng-if="!state.tagMode" type="text" class="form-control at-Input"
```

Comment:
the way this input is set up makes a bit more sense to me...standard input is given if tagMode is not passed.

Reply:
👍 That is good feedback to have, thank you. For what it is worth, the actual behavior of the textarea components discussed above is meant to be the same as it is here: The textarea should behave as a a standard textarea component unless `_tagMode` is passed.